Handlers/Phil.py

```diff
@@ -207,6 +207,11 @@
             "Most useful when also providing a reference geometry to xia2."
     .short_caption = "Fix geometry"
     .expert_level = 1
+  fix_distance = False
+    .type = bool
+    .help = "Whether or not to refine the detector distance in dials.index and dials.refine. " \
```

Comment:
```suggestion
    .help = "Do not refine the detector distance in dials.index and dials.refine."
```
gives clearer indication what `True` and `False` mean. Assume the trailing `\` is not relevant